tableaudocumentapi/multilookup_dict.py

```diff
@@ -40,6 +46,14 @@ def __setitem__(self, key, value):
 
         dict.__setitem__(self, key, value)
 
+    def get(self, key, default_value=AttributeError):
+        try:
+            return self[key]
+        except AttributeError:
+            if default_value != AttributeError:
```

Comment:
I got'cha.

I don't think a tuple is any cleaner. Sticking with the AttributeError seems fine